src/fs/driver/devfs/devfs_dvfs.c

```diff
@@ -287,3 +288,25 @@ static struct auto_mount devfs_auto_mount = {
 
 ARRAY_SPREAD_DECLARE(const struct auto_mount *const, auto_mount_tab);
 ARRAY_SPREAD_ADD(auto_mount_tab, &devfs_auto_mount);
+
+int devfs_add_block(struct block_dev *bdev) {
+	char full_path[DVFS_MAX_PATH_LEN];
```

Comment:
Seems like this function doesnt' create a new node in VFS tree.

Reply:
done